packages/ni.measurementlink.discovery.v1.client/src/ni/measurementlink/discovery/v1/client/_support.py

```diff
@@ -31,7 +31,8 @@
 
 
 def _get_discovery_service_address() -> str:
-    key_file_path = _get_key_file_path()
+    cluster_id = os.environ.get("NIDiscovery_ClusterId")
```

Comment:
nit: Environment variables are traditionally in UPPER_SNAKE_CASE. For examples, codesearch for `file:MeasurementServices GetEnv`